src/Features/Core/Portable/Diagnostics/Analyzers/NamingStyles/Serialization/SymbolSpecification.cs

```diff
@@ -199,10 +180,10 @@ private void PopulateModifierListFromXElement(XElement modifierListElement)
 
         private void PopulateCustomTagListFromXElement(XElement customTagListElement)
         {
-            RequiredCustomTagList = new List<string>();
+            var requiredCustomTagList = new List<string>();
             foreach (var customTag in customTagListElement.Elements("CustomTag"))
             {
```

Comment:
All references to the CustomTagList (this and others) are now removed (once I push)